lib/kinematics/src/rotation3.cpp

```diff
@@ -1,6 +1,6 @@
 // Copyright 2016 Chih-Chung Chou
-#include <iostream>
 #include <math.h>
```

Comment:
Re-order the include files according to the alphabet and the source.
The order is:
1. C/C++ std
2. 3rd party library
3. Our headers